### PR TITLE
Allow 'excludeFromIndexes' on listValues

### PR DESCRIPTION
**Pull request by SerafinTech**
Need to be able to turn off indexes on lists to prevent exploding indexes

### CODE REVIEW

**Comment by ryanseys** [comment]
Thanks for the PR. Looks like a good submission, just please lint the code with `npm run lint` before submitting :+1: and then your tests should pass here.

File: lib/datastore/request.js [hunk 1]
```diff
@@ -260,10 +260,17 @@ DatastoreRequest.prototype.save = function(entities, callback) {
           data.value = entity.valueToProperty(data.value);
 
           if (util.is(data.excludeFromIndexes, 'boolean')) {
-            data.value.indexed = !data.excludeFromIndexes;
+            
+            if(util.is(data.value.list_value, 'array')){
+              
+              for(var listValueIndex in data.value.list_value){
+                data.value.list_value[listValueIndex]['indexed'] = !data.excludeFromIndexes
+              }
```
Comment (stephenplusplus) [inline lib/datastore/request.js]: You can use our helper, `util.propAssign` here:

```js
var exclude = !data.excludeFromIndexes;
var list = data.value.list_value;

data.value.list_value = list.map(util.propAssign('indexed', exclude));
```

**Comment by stephenplusplus** [comment]
We took over the boring stuff: #430 :)

Thanks again!

**Comment by jlouthan** [comment]
Thanks for this! Will these updates be included in the version I get from `npm install gcloud` soon?

**Pull request closed without merging**